examples/spring-boot-minimal-kotlin/src/main/kotlin/example/springframework/boot/minimal/kotlin/ValidationExceptionHandler.kt

```diff
@@ -0,0 +1,46 @@
+package example.springframework.boot.minimal.kotlin
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.linecorp.armeria.common.HttpRequest
+import com.linecorp.armeria.common.HttpResponse
+import com.linecorp.armeria.common.HttpStatus
+import com.linecorp.armeria.common.MediaType
+import com.linecorp.armeria.server.ServiceRequestContext
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction
+import java.time.Instant
+import javax.validation.ValidationException
+
+/**
+ * A sample exception handler which handles a [ValidationException].
+ */
+class ValidationExceptionHandler : ExceptionHandlerFunction {
+
+    private val mapper = ObjectMapper()
+
+    override fun handleException(ctx: ServiceRequestContext, req: HttpRequest, cause: Throwable): HttpResponse {
+        return if (cause is ValidationException) {
+            try {
+                val status = HttpStatus.BAD_REQUEST
+                HttpResponse.of(status, MediaType.JSON,
+                        mapper.writeValueAsBytes(ErrorResponse(status.reasonPhrase(),
+                                cause.message ?: "empty message",
+                                req.path(),
+                                status.code(),
+                                Instant.now().toString())))
+            } catch (e: JsonProcessingException) {
+                HttpResponse.of(HttpStatus.INTERNAL_SERVER_ERROR, MediaType.PLAIN_TEXT_UTF_8,
+                        cause.message ?: "empty message")
+            }
+        } else ExceptionHandlerFunction.fallthrough()
+    }
+}
+
+/**
+ * A sample HTTP response which is sent to a client when a [ValidationException] is raised.
+ */
+data class ErrorResponse(val error: String,
```

Comment:
Don't we need [jackson-module-kotlin](https://github.com/FasterXML/jackson-module-kotlin) to convert `data class` to json?

Reply:
I added dependency [here](https://github.com/line/armeria/pull/2335/files#diff-d19512188cb37597acfa72d049a64f8aR25).

Reply:
Does it work without registering module?
```java
val mapper = ObjectMapper().registerModule(KotlinModule())
```

Reply:
Oh ... I didn't notice it. 😮 Thanks!

Each test passed, but I think I should check.

Reply:
I don't know the exact cause, but I fixed it as per the documentation guide. :sweat_smile: